azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/utilities/BottomCellItem.kt

```diff
@@ -4,8 +4,14 @@
 package com.azure.android.communication.ui.calling.utilities
 
 import android.graphics.drawable.Drawable
+import android.view.View
 import com.azure.android.communication.ui.calling.models.ParticipantViewData
 
+internal enum class BottomCellItemType {
+    BottomMenuAction,
```

Comment:
Not sure BottomMenuAction or BottomMenuItem, which one is better?

Reply:
Item is perhaps a bit too generic, Action implies that tapping will do something. I feel BottomMenuAction sounds more relevant.